src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * Class containing parsing methods, or conversions.
+ */
+public class Parser {
+    static final String TASK_TODO = " T ";
+    static final String TASK_EVENT = " E ";
+    static final String TASK_DEADLINE = " D ";
+
+
+    /**
+     * Parses date and time string and converts it into a LocalDateTime.
+     * @param dateTime string with date and time in format (dd/mm/yyyy hhmm).
+     * @return LocalDateTime object containing date and time.
+     */
+    public static LocalDateTime dateParser(String dateTime) {
```

Comment:
dateParser sounds more like a noun. Perhaps it is possible to make it sound more like a verb instead?